src/simplewallet/simplewallet.cpp

```diff
@@ -4468,6 +4485,23 @@ bool simple_wallet::show_transfer(const std::vector<std::string> &args)
       success_msg_writer() << "Timestamp: " << get_human_readable_timestamp(pd.m_timestamp);
       success_msg_writer() << "Amount: " << print_money(pd.m_amount);
       success_msg_writer() << "Payment ID: " << payment_id;
+      if (pd.m_unlock_time < CRYPTONOTE_MAX_BLOCK_NUMBER)
```

Comment:
But ffff.ffff is also valid, technically. I think I'll leave it as is. A chain rescan is all that's needed to get the correct values anyway.